### DISCUSSION TITLE
Detect All Objects as "other"

### DISCUSSION THREAD

**Discussion by mitchross** [blakeblackshear/frigate#2537] [Ask A Question] [answered]
Is it possible to create a catch all for "all objects" into say a other category? Meaning I have say 2 filters. dog and person, however I want to capture other object types in say a other folder so I can review other objects that triggered a record. Is this possible? For instance, hypothetical, what if there was a elephant in my back yard,but I didn't have object detection filtering for elephant in my config, I would miss the event capture.

**Comment by blakeblackshear** [comment, marked as answer]
You could rename those other objects to "other" via the config. You will probably just end up with a bunch of false positives and high CPU usage from frigate trying to track all of them. Many of those objects have very few examples in the training set.

**Reply by mitchross** [reply]
Thanks, didnt think about the cpu impact.
